ainow/templates/ainow/index.html

```diff
@@ -160,6 +160,7 @@
             <p class="person__job-title">Co-founder, University of Washington Tech Policy Lab, Assistant Professor, University of Washington School of Law</p>
         </div>
      </div>
+     <a href="schedule/workshop/attendees" class="btn btn-default homepage-people__full-link">All attendees and speakers</a>
```

Comment:
I've done that in  https://github.com/mysociety/ainow/pull/48/commits/627e87258469fd06b2a4066325716dba9d698c3b